src/lens/article/nodes/related_article/related_article.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+var Document = require('../../../substance/document');
+
+var RelatedArticle = function(node, document) {
+  Document.Node.call(this, node, document);
+};
+
+// Type definition
+// -----------------
+//
+
+RelatedArticle.type = {
+  "id": "related_article",
+  "parent": "content",
+  "properties": {
+    "source_id": "string",
+    "label": "string",
+    "articles": "array",
+  }
+};
+
+// Example Related Article
+// -----------------
+//
+
+RelatedArticle.example = {
+  "type": "related_article",
```

Comment:
related_articles and RelatedArticles are better names as they represent array of articles, not a single one